### DISCUSSION TITLE
Total power metrics for Node Level "kepler_node_joules_total"

### DISCUSSION THREAD

**Discussion by mukesh1327** [sustainable-computing-io/kepler#1598] [Q&A]
The metric `kepler_container_joules_total` provides overall total power consumption of pods by the sum of pkg, dram, uncore and gpu. But what about node level data. I can able to see the metrics **kepler_node_**, but there is no total power consumption value for nodes like container. Is it possible to get a new metric `kepler_node_joules_total` ?

**Comment by shuknk8s** [comment]
same here, could not find a way visualize node level power consumption metrics , the grafana dashboard shipped with kepler provides no info around power consumption/carbon foot print of nodes in the cluster
